I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/player_assessment.rpy", line 296, in script
    $ exam_weighted_score = (exam_score/10 * 0.2)
  File "game/player_assessment.rpy", line 296, in <module>
    $ exam_weighted_score = (exam_score/10 * 0.2)
NameError: name 'exam_score' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/player_assessment.rpy", line 296, in script
    $ exam_weighted_score = (exam_score/10 * 0.2)
  File "D:\installedApps\Ren'py\renpy-8.1.3-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\installedApps\Ren'py\renpy-8.1.3-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/player_assessment.rpy", line 296, in <module>
    $ exam_weighted_score = (exam_score/10 * 0.2)
NameError: name 'exam_score' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
CodeDreams 1.0
Sat Feb  3 22:04:38 2024
